addons/binding/org.openhab.binding.tesla/build.properties

```diff
@@ -0,0 +1,22 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/resty-0.3.2.jar,\
```

Comment:
please remove all the libs here, they do not seem to be required (btw, good job to not require any further libs!).